src/Compilers/Core/Portable/Operations/ControlFlowGraphBuilder.cs

```diff
@@ -775,7 +774,7 @@ private static bool PackBlocks(ArrayBuilder<BasicBlockBuilder> blocks, PooledDic
                                          next.Kind == ControlFlowBranchSemantics.Error ||
                                          next.Kind == ControlFlowBranchSemantics.ProgramTermination);
 
-                            IOperation value = block.FallThrough.Value;
+                            IOperation value = block.BranchValue;
```

Comment:
Can we assert that `!block.HasCondition` here?

Reply:
> Can we assert that !block.HasCondition here?

Yes we can, but that would simply duplicate condition of the ```if (!block.HasCondition)```, which is intended to separate conditional cases out. 

---
In reply to: [192563170](https://github.com/dotnet/roslyn/pull/27376#discussion_r192563170) [](ancestors = 192563170)

Reply:
Sorry, I'd missed that. Thanks